popolo/importers/popit.py

```diff
@@ -97,6 +97,15 @@ def update_optional_area(object_data):
                     area_id_to_django_object[area_id] = area
             return area
 
+        # Create all areas:
+        if 'areas' in data:
+            area_id_to_django_object = {}
+            for area_data in data['areas']:
+                with show_data_on_error('area_data', area_data):
+                    popit_id, area = \
```

Comment:
This is a tiny thing, but I'd call this variable `area_id` rather than `popit_id` - it's a bit clearer, and in the import-from-EP case, it's not a PopIt ID anyway :)